guides/components/component-arguments-and-html-attributes.md

```diff
@@ -1,144 +1,115 @@
-Components become useful building blocks of our app if we make them _reusable_. When we reuse components efficiently, we can avoid having to recreate parts of our app again and again. If you want to _reuse_ a component in multiple places, you'll need a way to template out parts of it.
+Les composants deviennent des éléments utiles d'une application quand on les rend "réutilisables". Lorsqu'on réutilise efficacement les composants, on peut éviter d'avoir à recréer des parties de l'app encore et encore. Si vous voulez "réutiliser" un composant à plusieurs endroits, vous aurez besoin d'un moyen d'en abstraire certaines parties.
 
-Let's start with two similar but not identical avatar components, that represent
-different users:
+Commençons par deux composants d'avatar similaires mais non identiques, qui représentent deux utilisateurs différents&nbsp;:
 
 ```handlebars {data-filename="app/components/received-message/avatar.hbs"}
 <aside>
-  <div class="avatar" title="Tomster's avatar">T</div>
+  <div class="avatar" title="Avatar de Tomster">T</div>
 </aside>
 ```
 
 ```handlebars {data-filename="app/components/sent-message/avatar.hbs"}
 <aside class="current-user">
-  <div class="avatar" title="Zoey's avatar">Z</div>
+  <div class="avatar" title="Avatar de Zoey">Z</div>
 </aside>
 ```
 
-The _structure_ of these components is identical, but they have somewhat
-different content (the user's first initial) and attributes (the `title` and `class`
-attributes).
+La "structure" de ces composants est identique, mais ils ont un contenu (l'initiale de l'utilisateur) et des attributs (`title` et `class`) différents.
 
 <div class="cta">
   <div class="cta-note">
     <div class="cta-note-body">
       <div class="cta-note-message">
-        You may notice that the <code>is-active</code> class on the
-        received message avatar from the previous chapters is missing here.
-        We'll cover that in the next chapter on
-        <a href="../conditional-content">Conditional Content</a>.
+        Vous avez peut-être noté la classe <code>is-active</code> sur l'avatar du message reçu, présente dans l'exemple des chapitres précédents, est absente dans celui-ci. Nous couvrirons ça dans le chapitre suivant, dans <a href="../conditional-content">Contenu Conditionnel</a>.
```

Comment:
"Vous avez peut-être noté la classe (...) est absente".
vs
"Vous avez peut-être noté que la classe (...) est absente".

```suggestion
        Vous avez peut-être noté que la classe <code>is-active</code> sur l'avatar du message reçu, présente dans l'exemple des chapitres précédents, est absente dans celui-ci. Nous couvrirons ça dans le chapitre suivant, dans <a href="../conditional-content">Contenu Conditionnel</a>.
```

Reply:
C'est une typo, thanks